src/test/java/com/google/cloud/compute/v1/it/ITComputeTest.java

```diff
@@ -51,41 +262,951 @@
+    assertNotNull(autoscalersScopedLists.iterator().next());
+  }
+
+  @Test
+  public void getBackendBucketTest() {
+    BackendBucket backendBucket = backendBucketClient.getBackendBucket(PROJECT_BACKEND_BUCKET_NAME);
+    assertEquals(BACKEND_BUCKET_NAME, backendBucket.getName());
+    assertEquals(BUCKET, backendBucket.getBucketName());
+    assertEquals(BACKEND_BUCKET_TARGET_LINK, backendBucket.getSelfLink());
+    assertEquals(Boolean.FALSE, backendBucket.getEnableCdn());
+  }
+
+  @Test
+  public void listBackendBucketsTest() {
+    BackendBucketClient.ListBackendBucketsPagedResponse backendBucketsList =
+        backendBucketClient.listBackendBuckets(PROJECT_NAME);
+    List<BackendBucket> backendBuckets = Lists.newArrayList(backendBucketsList.iterateAll());
+    assertThat(backendBuckets).isNotNull();
+    for (BackendBucket backendBucket : backendBuckets) {
+      if (BUCKET.equals(backendBucket.getBucketName())) {
+        assertEquals(BACKEND_BUCKET_NAME, backendBucket.getName());
+        assertEquals(BUCKET, backendBucket.getBucketName());
+        assertEquals(BACKEND_BUCKET_TARGET_LINK, backendBucket.getSelfLink());
+        assertEquals(Boolean.FALSE, backendBucket.getEnableCdn());
+      }
+    }
+  }
+
+  @Test
+  public void addResourcePoliciesDiskTest() {
+    DisksAddResourcePoliciesRequest resourcePoliciesRequest =
+        DisksAddResourcePoliciesRequest.newBuilder().build();
+    Operation resourcePoliciesDisk =
+        diskClient.addResourcePoliciesDisk(PROJECT_ZONE_DISK_NAME, resourcePoliciesRequest);
+    assertThat(resourcePoliciesDisk).isNotNull();
+    assertEquals("RUNNING", resourcePoliciesDisk.getStatus());
+    assertEquals("addResourcePolicies", resourcePoliciesDisk.getOperationType());
+    assertEquals(DISK_SELF_LINK, resourcePoliciesDisk.getTargetLink());
+    assertEquals(ZONE_SELF_LINK, resourcePoliciesDisk.getZone());
+  }
+
+  @Test
+  public void removeResourcePoliciesDiskTest() {
+    DisksRemoveResourcePoliciesRequest resourcePoliciesRequest =
+        DisksRemoveResourcePoliciesRequest.newBuilder().build();
+    Operation resourcePoliciesDisk =
+        diskClient.removeResourcePoliciesDisk(PROJECT_ZONE_DISK_NAME, resourcePoliciesRequest);
+    assertEquals("RUNNING", resourcePoliciesDisk.getStatus());
+    assertEquals("removeResourcePolicies", resourcePoliciesDisk.getOperationType());
+    assertEquals(DISK_SELF_LINK, resourcePoliciesDisk.getTargetLink());
+    assertEquals(ZONE_SELF_LINK, resourcePoliciesDisk.getZone());
+  }
+
+  @Test
+  public void aggregatedListDisksTest() {
+    DiskClient.AggregatedListDisksPagedResponse pagedListResponse =
+        diskClient.aggregatedListDisks(PROJECT_NAME);
+    List<DisksScopedList> disksScopedLists = Lists.newArrayList(pagedListResponse.iterateAll());
+    assertThat(disksScopedLists).isNotNull();
+    assertTrue(disksScopedLists.iterator().hasNext());
+    assertNotNull(disksScopedLists.iterator().next());
```

Comment:
instead of this, you can check the length. Because you are not verify anything concreate

Reply:
done

Reply:
done